tests/test_auto_parallel/test_tensor_shard/test_metainfo/test_activation_metainfo.py

```diff
@@ -104,5 +104,5 @@ def test_sofmax_meta_info():
 
 
 if __name__ == '__main__':
-    # test_ReLU_meta_concrete_info_match()
+    test_ReLU_meta_concrete_info_match()
```

Comment:
Please ename this function to test_relu_meta_concrete_info_match

Reply:
I decide to remove those changes and solve them in the next PR.